SPECS/httpd/CVE-1999-0236.nopatch

```diff
@@ -1 +0,0 @@
-# CVE-1999-0236 must be mitigated by the user. See "Server Side Includes" on https://httpd.apache.org/docs/2.4/misc/security_tips.html
```

Comment:
Note: These are being cleared due to the `autosetup` tool not handling non-empty .nopatch files cleanly.

Reply:
Thank you Daniel for the confirmation.